app/src/test/java/edu/uc/reedws/musiclink/PlaylistLocalDatabaseTest.kt

```diff
@@ -0,0 +1,23 @@
+package edu.uc.reedws.musiclink
+
+import org.junit.Assert.*
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.lifecycle.MutableLiveData
+import edu.uc.reedws.musiclink.dto.PlaylistDTO
+import edu.uc.reedws.musiclink.service.PlaylistService
+import edu.uc.reedws.musiclink.ui.main.MainViewModel
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+
+class PlaylistLocalDatabaseTest {
+
+    @Test
+    fun createAndSavePlaylist() {
+
```

Comment:
Empty unit test?

Reply:
Well he never showed how to test AndoirdViewModel so I couldn't write tests for this

Reply:
@devagrawal09 Can you add a TODO here then to explain why its blank?
```suggestion
// TODO: Write unit test for AndroidViewModel (ask prof for example)
```